### PR TITLE
Add chart icon, update graph-bar icon

### PR DESCRIPTION
**Pull request by kexin-zhang**
* Added a new chart icon 
* Replaced graph-bar SVG with updated SVG from PDS

image
image

### CODE REVIEW

File: packages/gestalt/src/icons/chart.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg"><path d="m10.7791936 13.2208064v-13.2208064c-6.05640271.62770002-10.7791936 5.74556089-10.7791936 11.9674117 0 6.6449969 5.38759139 12.0325883 12.0325883 12.0325883 6.2218508 0 11.3397117-4.7227909 11.9674117-10.7791936zm2.5067893-13.2208064c5.6442866.58458325 10.1294339 5.06973052 10.7140171 10.7140171h-10.7140171z" fill="#8e8e8e" fill-rule="evenodd"/></svg>
```
Comment (pfarejowicz) [inline packages/gestalt/src/icons/chart.svg]: @kexin-zhang are we sure these came from PDS designers? These normally dont have the `viewBox="0 0 24 24"` field specified if it came from them.

File: packages/gestalt/src/icons/graph-bar.svg [hunk 2]
```diff
@@ -1 +1 @@
-<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg"><path fill="#8e8e8e" d="M1 23V10h5v13H1zm8.5 0V1h5v22h-5zm8.5 0V7h4.999v16H18z"/></svg>
\ No newline at end of file
+<svg height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg"><path d="m13.5 1h-3c-.552 0-1 .448-1 1v20c0 .552.448 1 1 1h3c.552 0 1-.448 1-1v-20c0-.552-.448-1-1-1m8.5 6h-3c-.552 0-1 .448-1 1v14c0 .552.448 1 1 1h3c.552 0 1-.448 1-1v-14c0-.552-.448-1-1-1m-16 4v11c0 .552-.448 1-1 1h-3c-.552 0-1-.448-1-1v-11c0-.552.448-1 1-1h3c.552 0 1 .448 1 1" fill="#8e8e8e" fill-rule="evenodd"/></svg>
```
Comment (pfarejowicz) [inline packages/gestalt/src/icons/graph-bar.svg]: This also doesnt look minimized or optimized by PDS folks
image

File: packages/gestalt/src/icons/chart.svg (hunk 1 above)
Comment (kexin-zhang) [reply packages/gestalt/src/icons/chart.svg]: @pfarejowicz The designer I work with sent these over, and I believe they came from PDS. I'll double check and update here

**Comment by christianvuerings** [comment]
Closing due to inactivity, @kexin-zhang please re-open & address the comments if this is still an issue

**Pull request closed without merging**